app/services/report_importer.rb

```diff
@@ -95,7 +83,11 @@ def import_log_messages
     end
   end
 
-  def inspect_report
+  def report_status
+    ConfigReportStatusCalculator.new(:counters => raw['status']).calculate
```

Comment:
Chose to add a deprecation warning and return an empty hash. (https://github.com/theforeman/foreman/pull/2750/files#diff-5f2522f8937b1076938bcca2ab931cf1R88) Let me know if it's a valid solution